stacks-signer/src/config.rs

```diff
@@ -290,6 +298,10 @@ impl TryFrom<RawConfigFile> for Config {
             signer_id: raw_data.signer_id,
             signer_key_ids,
             event_timeout,
+            dkg_end_timeout: None,
```

Comment:
would this be better suited to have some default timeouts set?
what would happen if these config settings are not defined - i see them referenced in the runloop.rs file

Reply:
I think the only reasonable default is to not have timeouts.  We will likely have a lively debate over what timeouts we should use in prod as we approach release.  

Without timeouts, everything works properly, but users can block dkg or sign rounds forever if they are malicious.  So it's fine for testing to not have timeouts, but we need them for real world usage.

Reply:
understood, thanks!